### PR TITLE
Dash/dist-examples

### PR DESCRIPTION
**Pull request by goransle**
Adds new gulp task can be run with`npx gulp dashboards/dist-examples`. Requires [demo-manager](https://github.com/highcharts/highcharts-demo-manager).

Note that I did not have too much luck running the samples from the filesystem, which is why the output links to highcharts.com/samples.

### CODE REVIEW

File: tools/gulptasks/dist-examples.js [hunk 1]
```diff
@@ -204,12 +205,16 @@ async function createExamples(title, sourcePath, targetPath, template) {
...
+        sourcePath
+            .replaceAll('samples/', '')
+            .replaceAll('/demo', '')
+    );
 
     LogLib.success('Created', targetPath);
     const indexContent = localsidebar
-        .replace(/style=\"display:none;\"/g, '') // remove hidden style
-        .replace(/(?!href= ")(\.\/.+?)(?=")/g, 'examples\/$1\/index.html'); // replace links
+        .replaceAll('style="display:none;"', '') // remove hidden style
```
Comment (sebastianbochan) [inline tools/gulptasks/dist-examples.js:216]: Is it not saver to use regexp and set style as optional?

Comment (goransle) [reply tools/gulptasks/dist-examples.js:216]: Should be equivalent in this case. For context, this is the kind of file being replaced: https://www.highcharts.com/demo/sidebar.html

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Review by sebastianbochan — APPROVED** [review]

**Review by bre1470 — COMMENTED** [review]
I still not manage to test it. What target folder do I have to use? Can you maybe add a CLI help for `dist-examples`?

**Comment by bre1470** [comment]
@goransle helped with testing. Looks good. 🎉

**Review by bre1470 — APPROVED** [review]

**Pull request merged**